src/Engine/ProtoCore/Properties/Resources.resx

```diff
@@ -160,37 +160,37 @@
     <value>Converting an array to {0} would cause array rank reduction and is not permitted.</value>
   </data>
   <data name="kConvertDoubleToInt" xml:space="preserve">
-    <value>Converting double to int will cause possible information loss.</value>
+    <value>Converting a double to an int may lose information.</value>
   </data>
   <data name="kConvertNonConvertibleTypes" xml:space="preserve">
-    <value>Asked to convert non-convertible types.</value>
+    <value>Asked to convert between non-convertible types.</value>
```

Comment:
The object is being converted to non-convertible types, which will result null value.